tests/test_plots.py

```diff
@@ -29,23 +28,22 @@ def test_plot_all_one(data_dir):
     evening = [9.82, 9.34, 9.73, 9.93, 9.33, 9.41, 9.48, 9.14, 8.62, 8.60,
                9.60, 9.41, 8.43, 9.77, 8.96, 9.81, 9.75, 9.50, 9.90, 9.13]
 
-    filename = os.path.join(data_dir, 'results_one.bin')
+    filename = data_dir / 'results_one.cdf'
 
-    try:
-        with open(filename, "rb") as f:
-            best_out = pickle.load(f)
-            print('Previous analysis loaded from "%s"' % filename)
-    except (FileNotFoundError, EOFError, pickle.UnpicklingError):
+    if filename.exists():
+        model = best.BestModelOne(np.subtract(evening, morning), 0)
+        trace = InferenceData.from_netcdf(filename)
+        print('Previous analysis loaded from "%s"' % filename)
+        best_out = best.BestResultsOne(model, trace)
+    else:
         print("Performing Bayesian analysis")
         best_out = best.analyze_one(np.subtract(evening, morning))
-
         print('Saving results to "%s"' % filename)
-        with open(filename, "wb") as f:
-            pickle.dump(best_out, f)
+        best_out._trace.to_netcdf(filename)
```

Comment:
Let's use the read-only property here that's meant for public access:

```suggestion
        best_out.trace.to_netcdf(filename)
```